lua/neogit/popups/rebase/actions.lua

```diff
@@ -93,6 +94,24 @@ function M.interactively(popup)
   end
 end
 
+M.reword = operation("rebase_reword", function(popup)
+  local commit
+  if popup.state.env.commit then
+    commit = popup.state.env.commit
+  else
+    commit = CommitSelectViewBuffer.new(git.log.list()):open_async()[1]
+    if not commit then
+      return
+    end
+  end
+
+  -- TODO: Support multiline input for longer commit messages
+  local old_message = git.log.message(commit)
+  local new_message = input.get_user_input("Message: ", old_message)
```

Comment:
```suggestion
  local new_message = input.get_user_input("Message", old_message)
```

I cleaned up the `get_user_input()` function the other day, so the ": " at the end is added automatically.

Reply:
Ohh, just realized that we can't really tell the difference between "cancelled" and "empty input". Hmm.

Reply:
User can cancel just fine with ESC no? Then the whole rebase is never even started